packages/@coorpacademy-components/src/organism/accordion/container/index.js

```diff
@@ -22,6 +22,8 @@ const Accordion = props => {
           title={title}
           content={child}
           isOpen={isOpen}
+          moreIconType={moreIconType}
```

Comment:
use one icon. Apply a css rotation

Reply:
![screenshot from 2018-11-26 11-01-43](https://user-images.githubusercontent.com/45251186/48992719-fb9d8800-f16b-11e8-9422-a74c46e7a253.png)
Before change, this component imported 2 arrow. But in component I use only one arrow and apply a css for rotation arrow. I write to easily extend